src/mod.ts

```diff
@@ -32,18 +34,13 @@ export type EmojiFlavor<C extends Context = Context> = C & {
     ) => ReturnType<C["reply"]>;
 };
 
-function withEmoji(string: TemplateStringsArray, ...emojis: EmojiName[]) {
-    return string.reduce((acc, str, index) => {
-        // The number of elements in `string` is 1 more than in `emojis`, therefore
-        // we will be looking for an `undefined` emoji, which does not make sense,
-        // so we exit the loop early.
-        if (index === string.length - 1) return acc + str;
-        const emoji = getEmoji(emojis[index]);
-        return acc + str + (typeof emoji === "string" ? emoji : emoji.emoji);
+const getEmoji = (name: EmojiName) => emojis[name as keyof EmojiList] || name;
```

Comment:
Semantically, we want to replace absent values, not empty ones.
```suggestion
const getEmoji = (name: EmojiName) => emojis[name as keyof EmojiList] ?? name;
```